Header header
float64 latitude
float64 longitude
float64 altitude
float64 utm_easting
float64 utm_northing
string zone
string letter
int32 fix_quality
float64 hdop
string utc
string gngga_read
